lms/static/sass/discussion/_discussion.scss

```diff
@@ -71,7 +59,7 @@ body.discussion {
 
   .thread-title {
     display: block;
-    margin-bottom: $baseline;
+    margin-bottom: ($baseline);
```

Comment:
Haven't seen these parens around a sass variable before, curious if there's some issue here this is meant as a workaround for?